src/framework/theme/components/chat/chat-message.component.ts

```diff
@@ -81,12 +77,24 @@ import { NbChatMessageFile } from './chat-message-file.component';
                               [message]="message" [latitude]="latitude" [longitude]="longitude">
         </nb-chat-message-map>
 
+        <nb-chat-message-text *ngSwitchCase="'text'"
+                [sender]="sender" [date]="date" [dateFormat]="dateFormat"
+                [message]="message">
+        </nb-chat-message-text>
+
```

Comment:
Why we need to add `case='text'`? It looks like the same component will be used in the default case.

Reply:
Right it is not needed.